README.md

```diff
@@ -33,7 +33,7 @@ Also, you may have to make sure the storage node is in the network
 map.
 
 ``` sh
-$ docker exec -ti sn neofs-cli  control netmap-snapshot --binary-key /config/wallet-sn.key --endpoint 127.0.0.1:16513
+$ docker exec -ti sn neofs-cli  control netmap-snapshot --wallet /config/wallet-sn.key --endpoint 127.0.0.1:16513
```

Comment:
There is still a binary key inside, not a wallet. Will it auto-detect the content?

Reply:
Yes.

Reply:
3 separate mutually exclusive arguments were a bit confusing by the user experience. So we went back to the single argument that can auto detect the content. One day we will drop support of WIF and binary keys, I guess.

Reply:
Yes, remember the conversation now.